__tests__/app.test.js

```diff
@@ -22,4 +23,27 @@ describe("/api/categories", () => {
         });
       });
   });
+  test("GET - 200: responds with an array of categories each item in array have slug and decription keys", () => {
+    return request(app)
+      .get("/api/categories")
+      .expect(200)
+      .then((res) => {
+        res.body.categories.forEach((category) => {
+          expect(category).toEqual({
+            slug: expect.any(String),
+            description: expect.any(String),
```

Comment:
We need a length check here for this too. This forEach would also pass on an empty array since it would never be able to check an element and therefore fail it. 

I would also just condense the previous test into this one, we can check if its an array, if it has the correct length, and also that the elements are what we expected, all in one .then block here